lib/service_actor/core.rb

```diff
@@ -57,10 +63,22 @@ def fail!(**arguments)
 
   private
 
+  attr_accessor :argument_errors
```

Comment:
We should probably prefix these methods and instance methods with `actor` so as not to be overwitten by mistake inside user classes.

Reply:
Done: https://github.com/sunny/actor/pull/97/commits/cd67f44da23eefa78037b7ff5659ef629837bd77